pkg/ccl/backupccl/backup_test.go

```diff
@@ -1824,10 +1824,10 @@ func TestBackupRestoreResume(t *testing.T) {
 			t.Fatal(err)
 		}
 		backupDir := dir + "/backup"
-		if err := os.MkdirAll(backupDir, 0755); err != nil {
+		if err := os.MkdirAll(backupDir+"/"+progressDirectory, 0755); err != nil {
 			t.Fatal(err)
 		}
-		checkpointFile := backupDir + "/" + backupManifestCheckpointName
+		checkpointFile := backupDir + "/" + progressDirectory + "/" + backupManifestCheckpointName
```

Comment:
I don't know why this is the only line with spaces between the + and / but goland just keeps autocorrecting it to this (and everything else to no spaces).